kanidmd/server.toml

```diff
@@ -5,4 +5,4 @@ db_fs_type = "zfs"
 tls_ca = "../insecure/ca.pem"
 tls_cert = "../insecure/cert.pem"
 tls_key = "../insecure/key.pem"
-log_level = "perffull"
+log_level = "verbose"
```

Comment:
Is this the default configuration?

Reply:
The server.toml here is meant for testing really, so maybe I should set this to the maximum level instead?

Reply:
Yep, it makes sense.